scripts/build.js

```diff
@@ -27,16 +27,15 @@ getPackageNames(packageNames => {
       'exec',
       '--scope',
       scopeGlob,
-      '--ignore',
-      '@tds/shared-*',
+      '--no-private',
```

Comment:
Will this no longer create version bumps for private packages?

Reply:
https://www.npmjs.com/package/@lerna/filter-options#--no-private

`@tds/shared-*` components used to be private packages that were used in tds-core.

If my understanding is correct, now that we are publishing components with the `@tds/shared-*` prefix we can no longer ignore them, instead we can use [`--no-private`](https://www.npmjs.com/package/@lerna/filter-options#--no-private) to exclude all private packages.

Reply:
I see, so this only excludes private packages from **build**, and none of them have a rollup config, so they wouldn't build anyway.